README.md

```diff
@@ -95,6 +95,20 @@ $ docker-compose up
 
 Agora basta acessar o navegador em [localhost:8000](http://localhost:8000) para ver o conteúdo gerado.
 
+Instalando e Rodando no Windows (Docker)
+--------------------------
+- [Opcional] Instale o [Visual Studio Code](https://code.visualstudio.com/) para fazer códigos legais;
+- [Opcional mas fortemente indicado] Instale o [Git para Windows](https://desktop.github.com/) para um shell mais legal também;
+- Python 3.8 está disponível na loja do Windows e você deve instalar também. Só procurar e colocar em obter que está tudo certo;
+- Abra o Windows Powershell como administrador e faça a instalação do [chocolatey](https://chocolatey.org/install). Com ele poderemos instalar o comando make que será utilizado junto ao Docker;
+- Com o comando *choco* sendo reconhecido no Windows, [instale o make](https://chocolatey.org/packages/make) com `choco install make`;
+- Por último, faça a instalação do [Docker](https://docs.docker.com/docker-for-windows/install/), certifique-se que os requisitos mínimos estão sendo cumpridos. Para o Windows 10 Home, recomendo que faça uma atualização do sistema antes da instalação (Configurações → Atualização e Segurança → Windows Update)
+    - Atente-se se o WSL2 está rodando na sua máquina. Se ainda for o WSL, [atualize](https://docs.microsoft.com/pt-br/windows/wsl/wsl2-kernel).
+- Faça fork do [repositório](https://github.com/pyladies-brazil/br-pyladies-pelican);
+- Reinicie o computador para garantir que todas as mudanças foram efetuadas e salvas;
+- Abra o projeto no VSCode e rode `make up`.
```

Comment:
uma dúvida, quando você faz o make up pelo VSCode, ele que sobe o docker?

Reply:
pq se for esse o caso, depois que a pessoa instala o docker no windows, seria só usar o `docker compose-up', não?

Reply:
o Windows sobe o Docker. daora.